### PR TITLE
added like functionality

### PR DESCRIPTION
**Pull request by sushmita12345**
Added like functionality in Video Library

Screenshot 2022-05-21 at 9 24 48 PM

### CODE REVIEW

**Comment by anandsr21harshit** [comment]
- UX for liking video is not good.
- On clicking like video I can't differentiate whether it is clicked or not

Suggestion-
1. On clicking on like button it's color should changed and the content should also change from "like" to "liked"
2. Upon clicking on "liked" it should remove the video from liked videos and the content of button should change to "like"
3. Add a hover effect: changing pointer to cursor

File: src/pages/LikePage/LikePage.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Sidebar, LikeCard } from "../../components/index";
+import {useLike} from "../../context/LikeContext"
+
+export function LikePage() {
+
+    const {likeState: {likedVideo}} = useLike()
+    // console.log(likedVideo)
```
Comment (anshulraheja) [inline src/pages/LikePage/LikePage.jsx]: Remove the comments. The code structure is good, just need to work on the UI of the like button

**Review by anshulraheja — APPROVED** [review]

**Review by palash23jain — APPROVED** [review]
code looks clean, good to be merged

**Comment by sushmita12345** [comment]
Changes has been done, thank you.

**Review by anandsr21harshit — APPROVED** [review]

**Pull request merged**